### PR TITLE
Promote Robb Kidd from Approver to Maintainer

### PR DESCRIPTION
**Pull request by robbkidd**
More hands on deck for merging and releasing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The Ruby special interest group (SIG) meets regularly. See the OpenTelemetry
 
 Approvers ([@open-telemetry/ruby-contrib-approvers](https://github.com/orgs/open-telemetry/teams/ruby-contrib-approvers)):
 
-- [Robb Kidd](https://github.com/robbkidd), Honeycomb
+- (Could _your_ name could appear here?)
```
Comment (robbkidd) [inline README.md:30]: Felt odd to leave it empty and also odd to remove the section entirely. What's here is intended to welcome new contributors and it is certainly open to word-smithing feedback and alternatives.

**Comment by robbkidd** [comment]
I sure hope rdkafka doesn't hold me back! 

nervous Kermit

**Review by ericmustin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (robbkidd) [reply README.md:30]: Ugh. "Could" appears twice in this sentence. What even are words?

**Review by plantfansam — COMMENTED** [review]
WOOOO!

**Pull request merged**